types/sm-crypto/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for sm-crypto 0.1
+// Project: https://github.com/JuneAndGreen/sm-crypto#readme
+// Definitions by: Thermod <https://github.com/me>
```

Comment:
this is a default placeholder, needs to be amended witt your username:
```ts
// Definitions by: Thermod <https://github.com/Moonisky>
```

Reply:
Sorry for that, it's my mistake.